src/test/java/com/karankumar/bookproject/ui/components/utils/ComponentUtilTest.java

```diff
@@ -0,0 +1,22 @@
+package com.karankumar.bookproject.ui.components.utils;
+
+import com.vaadin.flow.component.HasValue;
+import com.vaadin.flow.component.textfield.TextField;
+import org.junit.jupiter.api.Test;
+
+import static org.mockito.Mockito.*;
+
+class ComponentUtilTest {
+
+    @Test
+    void shouldClearComponentFields() {
```

Comment:
Could you please add few more tests:
1. Don't clear if the component is null
2. Don't clear if the component is empty
3. Don't clear if there is no component in the list

Reply:
I have some more unit test, could you please take a look.
Thanks